lib/kube/proxy/forwarder.go

```diff
@@ -2770,3 +2776,25 @@ func errorToKubeStatusReason(err error, code int) metav1.StatusReason {
 		return metav1.StatusReasonUnknown
 	}
 }
+
+// formatForwardResponseError formats the error response from the connection
+// monitor to a Kubernetes API error response.
+type formatForwardResponseError func(status *apierrors.StatusError) error
+
+func (f formatForwardResponseError) WriteString(s string) (n int, err error) {
+	if f == nil {
+		return len(s), nil
+	}
+	_ = f(
```

Comment:
Why not return the error, as supported by the signature?

If we are going to ignore it, then let's at least log the error.